scipy/stats/tests/test_stats.py

```diff
@@ -4648,12 +4654,20 @@ def test_normalitytests():
     assert_array_almost_equal(stats.kurtosistest(x, nan_policy='omit'),
                               expected)
 
+    # test alternative with nan_policy='omit'
```

Comment:
These are OK if we assume that `alternative='two-sided'` with `nan_policy='omit'` is well-tested. Do you think that is the case?

Reply:
> Do you think that is the case?

Yes, I see a test for that case for both `skewtest` and `kurtosistest`. I can remove these then if you want.

Reply:
No, I don't mean to remove these. I'm saying that the test is only valid if we trust that `mstats.skewtest` with `nan_policy='omit'` does the right thing. 

I just don't think that the existing test of `mstats.skewtest` with `nan_policy='omit'` is sufficient. But I don't think that is something you have to fix in this PR, so never mind.